resilience4j-circuitbreaker/src/main/java/io/github/resilience4j/circuitbreaker/internal/Token.java

```diff
@@ -0,0 +1,15 @@
+package io.github.resilience4j.circuitbreaker.internal;
+
+import java.time.LocalDateTime;
+
+public class Token {
+    private LocalDateTime timestamp;
```

Comment:
I think we should use `System.nanoTime()` and  `long` as the creation timestamp in order to avoid too many objects.
It could be created inside of the constructor, or?